2023/f8995.py

```diff
@@ -0,0 +1,30 @@
+from form import Form, FilingStatus
+
+class F8995(Form):
+    """Form 8995A, Qualified Business Income Deduction Simplified Computation"""
+    def __init__(f, inputs, f1040, sched_d, sse):
+        super(F8995, f).__init__(inputs)
+
+        if inputs['status'] == FilingStatus.JOINT:
+            f['2']  = sse[0]['3'] - sse[0]['13'] if sse[0].mustFile else 0
+            f['2'] += sse[1]['3'] - sse[1]['13'] if sse[1].mustFile else 0
+        else:
+            f['2']  = sse['3'] - sse['13'] if sse.mustFile else 0
+        f['3'] = inputs.get('qualified_business_net_carryforward')
+        f['4'] = max(0, f['2'] - f['3'])
```

Comment:
I think it's best to express this as a sum `max(0, f['2'] + f['3'])` so that losses are negative numbers.